browser/themes/brave_dark_mode_utils.cc

```diff
@@ -76,135 +16,4 @@ void RegisterBraveDarkModeLocalStatePrefs(PrefRegistrySimple* registry) {
       static_cast<int>(BraveDarkModeType::BRAVE_DARK_MODE_TYPE_DEFAULT));
```

Comment:
why do we still have references to brave dark mode here?

Reply:
Only for migration.
We need to migrate current dark mode type to browser color scheme.